railties/test/generators/plugin_generator_test.rb

```diff
@@ -226,6 +226,21 @@ def test_skip_javascript
     end
   end
 
+  def test_skip_action_mailer_with_mountable
+    run_generator [destination_root, "--mountable", "--skip-action-mailer"]
+    assert_no_directory "app/mailers"
```

Comment:
Hmm we're already testing that the `app/mailers` directory does not exists in 

https://github.com/rails/rails/blob/bcfa51fba6ee529fbe357a183fd5bc8bebf13513/railties/test/generators/shared_generator_tests.rb#L304-L305

Any ideas why that test is passing?

Reply:
The test specified neither `--full` option nor `--mountable` option, so the `app` folder didn't create at all.

Reply:
Got it :) Do we have to ensure that the `dummy/app/mailers` directory is not created too?

Reply:
Oops sorry I was testing without `--skip-action-mailer`. Looks like it does remove the `test/dummy/app/mailers` directory.